### PR TITLE
fix(vps): fix translations in veam and snapshot order

### PR DESCRIPTION
**Pull request by JDownloader**
Please don't merge until we have a plan for those two translations.

### CODE REVIEW

File: client/app/vps/snapshot-order/vps-snapshot-order.html [hunk 1]
```diff
@@ -59,12 +59,12 @@
...
                     </dl>
                     <div data-ng-show="$ctrl.model.url" class="row-fluid">
-                        <span data-translate="vps_configuration_veeam_order_step2_bc"></span>
-                        <span class="blue bold"
-                              data-translate="vps_configuration_veeam_order_step2_continue"></span>
-                        <span data-translate="vps_configuration_veeam_order_step2_explication"></span>
-                        <span class="fs11 lh11 italic"
-                              data-translate="vps_configuration_veeam_order_step2_explication2"></span>
+                        <div data-translate="vps_configuration_veeam_order_step2_bc"></div>
+                        <div class="blue bold"
```
Comment (FredericEspiau) [inline client/app/vps/snapshot-order/vps-snapshot-order.html]: Oh

File: client/app/vps/snapshot-order/vps-snapshot-order.html [hunk 2]
```diff
@@ -59,12 +59,12 @@
...
+                             data-translate="vps_configuration_veeam_order_step2_continue"></div>
+                        <div data-translate="vps_configuration_veeam_order_step2_explication"></div>
+                        <div class="fs11 lh11 italic"
```
Comment (FredericEspiau) [inline client/app/vps/snapshot-order/vps-snapshot-order.html]: Gawd

File: client/app/vps/translations/Messages_fr_FR.xml [hunk 3]
```diff
@@ -318,13 +318,13 @@
...
+   <translation id="vps_configuration_veeam_order_step1_id">Afin de confirmer votre demande, veuillez accepter les conditions d'utilisation.</translation>
    <translation id="vps_configuration_veeam_order_step1_totalHT" qtlid="169238">Prix HT :</translation>
    <translation id="vps_configuration_veeam_order_step1_totalTTC" qtlid="169251">Prix TTC :</translation>
    <translation id="vps_configuration_veeam_order_step1_contracts" qtlid="170070">Je reconnais avoir bien pris connaissance des éléments suivants :</translation>
    <translation id="vps_configuration_veeam_order_step2_generation_bc" qtlid="169511">Génération du bon de commande ...</translation>
    <translation id="vps_configuration_veeam_order_step2_bc" qtlid="169524">Le bon de commande a été généré avec succès.</translation>
-   <translation id="vps_configuration_veeam_order_step2_continue" qtlid="169537">Afin de confirmer votre commande, veuillez cliquer sur le bouton "Régler" ci-dessous.</translation>
+   <translation id="vps_configuration_veeam_order_step2_continue">Afin de confirmer votre commande, veuillez cliquer sur le bouton "{{:: 'common_confirm' | translate }}" ci-dessous.</translation>
```
Comment (FredericEspiau) [inline client/app/vps/translations/Messages_fr_FR.xml]: This actually works ? 😮

Comment (JDownloader) [reply client/app/vps/translations/Messages_fr_FR.xml]: Last time I checked yes, It's probably awesome for translators (can't missmatch button name, but we shouldn't over do it

File: client/app/vps/snapshot-order/vps-snapshot-order.html (hunk 2 above)
Comment (JDownloader) [reply client/app/vps/snapshot-order/vps-snapshot-order.html]: ermahgerd, yes

**Comment by jleveugle** [comment]
Hello @JDownloader,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**